bin/yarn.js

```diff
@@ -5,14 +5,12 @@
 'use strict';
 
 // init roadrunner
-var userHome = require('user-home');
 var mkdirp = require('mkdirp');
-var path = require('path');
-var CACHE_FILENAME = path.join(userHome, '.yarn', '.roadrunner.json');
-mkdirp.sync(path.dirname(CACHE_FILENAME));
+var constants = require('../lib-legacy/constants');
```

Comment:
I now also require constants here from lib-legacy, so that I can use the folder constants. I think that shouldn't be a significant problem.